src/components/MultiHopTrade/components/TradeInput/components/ManualAddressEntry.tsx

```diff
@@ -1,5 +1,6 @@
 import { FormControl, FormLabel, Link } from '@chakra-ui/react'
 import { isLedger } from '@shapeshiftoss/hdwallet-ledger'
+import { isMetaMask } from '@shapeshiftoss/hdwallet-metamask'
```

Comment:
That's fine for now as an improvement but we'll want to consume `checkIsMetaMaskDesktop` instead, or this will also display for MM mobile, and all MM impersonators, for which the MM multichain snap is unavailable.